setup.py

```diff
@@ -2,12 +2,15 @@
 
 from distutils.extension import Extension
 from Cython.Distutils import build_ext
+# to get the proper numpy header locations (if using anaconda)
+import numpy as np
 
 ext_modules = []
 ext_modules.append(Extension(
```

Comment:
That is reasonable, but, given your earlier comments about "I thought no
one was using it," perhaps not worth the effort currently!

On Tue, Jul 26, 2016 at 2:20 PM, Eric Hunsberger <notifications@github.com>
wrote:

> In setup.py <https://github.com/hunse/kitti/pull/3#discussion_r72307343>:
>
> > @@ -2,12 +2,15 @@
> >
> >  from distutils.extension import Extension
> >  from Cython.Distutils import build_ext
> > +# to get the proper numpy header locations (if using anaconda)
> > +import numpy as np
> >
> >  ext_modules = []
> >  ext_modules.append(Extension(
>
> What would you think about moving the BP stuff out into a separate
> repository? To me, it just doesn't really seem to fit here because it's not
> specific to the KITTI dataset at all.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/hunse/kitti/pull/3/files/0c730a117cf28087a62e954662741363dd4b6014#r72307343>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQLpJpxKDbfQfYlz1HtlRMIgXa7wQQonks5qZk_ogaJpZM4JL2GG>
> .
>